.github/workflows/ci.yml

```diff
@@ -27,10 +28,30 @@ jobs:
         run: |
           python -m pip install --upgrade pipenv
           pipenv install --dev
+          mkdir "${RUNNER_TEMP}/bin"
+          curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b "${RUNNER_TEMP}/bin"
+          chmod +x "${RUNNER_TEMP}/bin/syft"
+          curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sh -s -- -b "${RUNNER_TEMP}/bin"
+          chmod +x "${RUNNER_TEMP}/bin/grype"
+          echo "${RUNNER_TEMP}/bin" >> "${GITHUB_PATH}"
       - name: Run tests
         run: |
           find tests -mindepth 1 -type d -exec chmod o+w {} \;
-          pipenv run invoke test --debug
+          pipenv run invoke test
+      - name: Generate SBOMs
+        run: pipenv run invoke sbom
+      - name: Upload SBOMs
```

Comment:
This and the following step gives us SBOM and vuln scanning artifacts for commits of an open PR (if multiple are pushed at once, only the last one will trigger a github actions run).